Main.java

```diff
@@ -1,15 +1,29 @@
-/**
+import java.util.Scanner;
+
+/** 
  *
- * @author 
+ * @author Gavin Sandhar

```

Comment:
Don't forget the description in the code header. It should appear on line 4 in your code.